content/en/docs/contribute/generate-ref-docs/prerequisites-ref-docs.md

```diff
@@ -5,18 +5,17 @@
 
 - You need to have these tools installed:
 
-  - [Python](https://www.python.org/downloads/) v3.7.x
+  - [Python](https://www.python.org/downloads/) v3.7.x+
   - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-  - [Golang](https://golang.org/doc/install) version 1.13+
+  - [Golang](https://go.dev/dl/) version 1.13+
   - [Pip](https://pypi.org/project/pip/) used to install PyYAML
   - [PyYAML](https://pyyaml.org/) v5.1.2
   - [make](https://www.gnu.org/software/make/)
   - [gcc compiler/linker](https://gcc.gnu.org/)
-  - [Docker](https://docs.docker.com/engine/installation/) (Required only for `kubectl` command reference)
+  - [Docker](https://docs.docker.com/engine/installation/) (Required only for `kubectl` command reference, also kubernetes moving on from dockershim read [here](https://kubernetes.io/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/))
```

Comment:
We prefer to use site-relative links where feasible.

Reply:
Hi @dipesh-rawat, updated the PR.